social.md

```diff
@@ -26,11 +26,11 @@ List of channels actively maintained by JMS in alphabetical order:
 - Medium: https://blog.jupyter.org
 - Mastodon:  https://hachyderm.io/@ProjectJupyter
 - Slack JupyterLab: https://jupyterlabworkspace.slack.com
-- X/Twitter Jupyter: https://twitter.com/ProjectJupyter
 - YouTube: https://youtube.com/@projectjupyter
+- BlueSky : https://bsky.app/profile/projectjupyter.bsky.social
```

Comment:
oops, BlueSky was already at the top of this list, now we have a bluesky sandwich. :D